testing/spectest/mainnet/altair/epoch_processing/epoch_processing_test.go

```diff
@@ -0,0 +1,13 @@
+package epoch_processing
+
+import (
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/config/features"
+)
+
+func TestMain(m *testing.M) {
```

Comment:
What is this? I remember @prestonvanloon and I saw something similar back in `hf1` branch and decided to delete it

Reply:
This basically runs all tests in that package with those flags pre-loaded. Since this touches HTR, I wanted to run them through spec tests here.

Reply:
Any reason it has to call `TestMain` than an explicit name like `TestEpochProcessWithBalanceTrie` ?

Reply:
oh 
https://pkg.go.dev/testing#hdr-Main

This basically allows for test setup and cleanup rather than doing it per test.